cob_hardware_config/cob4-3/config/base_driver.yaml

```diff
@@ -0,0 +1,54 @@
+defaults:
+  eds_pkg: cob_hardware_config
+  eds_file: "cob4-3/config/Elmo.dcf"
+  dcf_overlay:
+    "6083": "1000000"
+    "6084": "1000000"
+    "60C5": "1000000"
+    "60C6": "1000000"
+    "6099sub1": "100000" # find switch (coarse)
+    "6099sub2": "10000" # find home (fine)
+    "1016sub1" : "0x7F0064" # heartbeat timeout of 100 ms for master at 127
+
+drive_wheel: &drive_wheel
+  dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+    "6098": "35" # homing method
+  vel_from_device: "v != v ? v=0 : v=smooth(deg2rad(obj606c/1000.0),v,0.3)" # for smoother current velocity in joint_states
+
+nodes:
+  ##Wheel 1
+  fl_caster_rotation_joint:
+    id: 1
+    dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+      "6098": "19" # homing method
+      "607C": "-51445"#-52502 -54359 home offset
+  fl_caster_r_wheel_joint:
+    <<: *drive_wheel
+    id: 2
+   ## dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+   ##   "6098": "35" # homing method
+
+  ##Wheel 2
+  b_caster_rotation_joint:
+    id: 3
+    dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+      "6098": "19" # homing method
+      "607C": "-174876.0" #-172553.0 -175428 home offset
+  b_caster_r_wheel_joint:
+    <<: *drive_wheel
+    id: 4
+  ##  dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+  ##    "6098": "35" # homing method
+
+  ##Wheel 3
+  fr_caster_rotation_joint:
+    id: 5
+    dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+      "6098": "19" # homing method
+      "607C": "67570.5" #65675.5 -111597 home offset
```

Comment:
previous values should be retrieved through github-history....having them in comments is rather confusing...